courses/fundamentals_of_ada/065_type_derivation.rst

```diff
@@ -336,17 +321,17 @@ Implicit Subtype
 
    .. code:: Ada
 
-      type <Anon> is new Predefined_Integer_Type;
-      subtype T is <Anon> range L .. R;
+      type T is new Predefined_Integer_Type;
+      subtype T is T range L .. R;
```

Comment:
Same name for type and subtype (call the first one "T_Prime?")

Reply:
Done